src/main/java/mate/academy/springboot/datajpa/service/mapper/DtoMapper.java

```diff
@@ -0,0 +1,7 @@
+package mate.academy.springboot.datajpa.service.mapper;
+
+public interface DtoMapper<R, M, S> {
```

Comment:
maybe it's enough to have only 2 generics instead of 3? You need to make from M to R and vice versa